staging/src/k8s.io/apiserver/pkg/storage/value/transformer.go

```diff
@@ -32,6 +32,14 @@ type Context interface {
 	AuthenticatedData() []byte
 }
 
+// KMSService allows encrypting and decrypting data using an external Key Management Service.
```

Comment:
I don't think this is the right place to put this. There doesn't appear to be anything in this package using this interface. The interface should be co-located with the code that accepts interface.